rubin_sim/maf/run_selfcal_metric.py

```diff
@@ -61,7 +72,7 @@ def run_selfcal_metric():
 
     # Make plots of the residuals map
     map_bundle = MetricBundle(
-        IdentityMetric(metric_name="PhotoCal Uniformity"),
+        IdentityMetric(metric_name="PhotoCal Uniformity, %s" % args.filter),
```

Comment:
Just to make things print nicer in the end, can you drop the ","? 
I admit this is just a matter of personal taste but  "Photocal Uniformity r <>" (and presumably maybe other bands ..)  just reads a bit easier than "Photocal Uniformity, r <>" ... 
I guess I don't care a ton but none of the other summary metrics use ","